docs/scripts_md/minc_deletion.md

```diff
@@ -0,0 +1,54 @@
+# NAME
+
+minc\_deletion.pl -- this script deletes files records from the database, and
+archives the actual files. Files to be deleted can be specified either based on
```

Comment:
replace "actual files" by "backend files stored in /data/project/data"

Reply:
ok! I added assembly in the end since we are deleting MINC files.